packages/wonder-blocks-link/components/link.js

```diff
@@ -100,6 +100,23 @@ export type SharedProps = {|
      * stubbed out.
      */
     onClick?: (e: SyntheticEvent<>) => mixed,
+
+    /**
+     * Run async code before navigating. If the promise returned rejects then
```

Comment:
I think it would be useful to be explicit about "navigating" here referring to navigating to the URL passed in to the `href` prop.